### DISCUSSION TITLE
Installing 8.0.11 hosting bundle is not installing all components

### DISCUSSION THREAD

**Discussion by nagababu0579** [dotnet/AspNetCore.Docs#34247] [General]
Until 8.0.7, on installing hosting bundle, we can see all components like windows server hosting, .net runtime and asp.net core shared framework. But when tried installing 8.0.11, it does not list components like .net runtime and asp.net core shared framework in control panel. Any changes happened recently after 8.0.7 hosting bundle versions? I have a workaround to install required components separately, but would like to know the reason why it does not show them in control panel or do not install components like .net runtime?

**Comment by gfoidl** [comment]
A bit related: https://github.com/dotnet/aspnetcore/issues/59202
